nvm.sh

```diff
@@ -2251,6 +2239,43 @@ nvm_download_artifact() {
   nvm_echo "${TARBALL}"
 }
 
+# args: nvm_os, version, tarball, tmpdir
+nvm_extract_tarball() {
+  local NVM_OS
```

Comment:
let's check for the argument count here, and fail if it's not exactly 4

Reply:
Updated PR to add that check.